go/server/server.go

```diff
@@ -0,0 +1,22 @@
+package main
```

Comment:
We should probably move `go/server/server/go` to `go/cmd/server.go` as per https://github.com/golang-standards/project-layout/blob/master/cmd/README.md